src/atoms/SegmentControl/SegmentControlOption.native.js

```diff
@@ -0,0 +1,239 @@
+import React, { useCallback, useContext } from 'react';
+import { TouchableHighlight } from 'react-native';
+import PropTypes from 'prop-types';
+import styled, { ThemeContext } from 'styled-components/native';
+import Text from '../Text';
+import { useSegmentControlContext } from './SegmentControlContext';
```

Comment:
`useSegmentControlContext`should be after `./../_helpers/automation-attributes`. You can rebase with master as @kamleshchandnani has added support to organise the imports in vs code.